### PR TITLE
Add missing keyword argument to tqdm

### PR DESCRIPTION
**Pull request by NeoPlato**
The part that shows the animations is currently unavailable.

Screenshot 2020-12-30 195301

With one line of code, that can be fixed.

### CODE REVIEW

**Comment by NeoPlato** [comment]
Question, is there the off chance we could also slice the description.

Cause of things like this:

image

**Comment by leotrs** [comment]
> With one line of code, that can be fixed.

Sounds like an easy PR :wink:

**Review by kilacoda — APPROVED** [review]

**Review by leotrs — APPROVED** [review]

**Comment by leotrs** [comment]
> > With one line of code, that can be fixed.
> 
> Sounds like an easy PR 😉

Lol when I said this I thought this was an issue, not a PR

**Comment by NeoPlato** [comment]
> Lol when I said this I thought this was an issue, not a PR

Oh no I always provide the solution when I can.

Before this gets merged though, would it also be a good idea to change the string representation of animations to be the class name of the animation plus the _name_ of the mobject.

Also one line of code

**Comment by leotrs** [comment]
> Before this gets merged though, would it also be a good idea to change the string representation of animations to be the class name of the animation plus the name of the mobject.

I'd rather open a new PR for that, especially because I'd like to see some documentation about naming mobjects - which would make it more than a one line change... (I have personally never once named a mobject and I believe this is not documented anywhere).

**Comment by NeoPlato** [comment]
There really isn't that much to naming an mobject (most of the time the name comes from subclassing existing ones for added functionality) so the illusion of naming it in that fashion may actually be redacted.

The lengthy string representation of VGroup was what I was mildly concerned with. But it seems like we could move that as the `__repr__` attribute then simply use `__str__` as the class name for all mobjects.

Yeah that seems like a sensible PR

**Comment by behackl** [comment]
I am not sure this should be merged in its current state: it makes RTD log files much harder to read due to the extremely long string representation of some Animations, as has already been mentioned: Compare https://readthedocs.org/api/v2/build/12661396.txt (this branch) with https://readthedocs.org/api/v2/build/12742870.txt (a recent build of `master` ).

**Comment by NeoPlato** [comment]
> due to the extremely long string representation of some Animations

The lengthy representation of animations actually comes from lengthy representations of some mobjects, especially VGroup. I really should've gotten to that other PR with that fix, but I can also attach that fix here.

Except I've gotten a PR nit-picked in the past so I have to know when I step out of convention

**Comment by behackl** [comment]
> > due to the extremely long string representation of some Animations
> 
> The lengthy representation of animations actually comes from lengthy representations of some mobjects, especially VGroup. I really should've gotten to that other PR with that fix, but I can also attach that fix here.

If you have a good approach how to shorten (maybe just by actually limiting?) the string representation of these VGroups (personally, I'd go for a representation like `VGroup consisting of 42 mobjects` or something like that), this is very welcome.

> 
> Except I've gotten a PR nit-picked in the past so I have to know when I step out of convention

An excellent trigger for my bad consience! I am sorry about that, but I'll revisit #778 soon; I hope to be able to offer a good way forward.

**Comment by NeoPlato** [comment]
> If you have a good approach how to shorten (maybe just by actually limiting?) the string representation of these VGroups

Piggybacking on existing implementation, yes, quite easy. It is now as I described it. The lengthy representation goes to `__repr__` to be accessed by those who care. But implicit string representation defaults to `__str__` so this should probably work

> An excellent trigger for my bad consience! I am sorry about that, but I'll revisit #778 soon; I hope to be able to offer a good way forward.

I could easily forget this exists sometimes. But it is kinda code heavy and I tied my hands on new development by making a PR of it from my master branch so that's understandable

File: manim/mobject/types/vectorized_mobject.py [hunk 1]
```diff
@@ -1029,6 +1029,11 @@ def __repr__(self):
             + ")"
         )
 
+    def __str__(self):
+        return (
+            f"self.__class__.__name__({len(self.submobjects)})"
```
Comment (behackl) [inline manim/mobject/types/vectorized_mobject.py:1034]: Currently, this produces output like:

```py
Animation 1: Transformself.__class__.__name__(484):   0%|          | 0/30 [00:00<?, ?it/s]
Animation 1: Transformself.__class__.__name__(484):   3%|▎         | 1/30 [00:00<00:16,  1.75it/s]
Animation 1: Transformself.__class__.__name__(484):   7%|▋         | 2/30 [00:01<00:15,  1.80it/s]
```

so I don't think we are there yet.

```suggestion
            f"{self.__class__.__name__} of {len(self.submobjects)} submobjects"
```

**Review by behackl — CHANGES_REQUESTED** [review]

File: manim/mobject/types/vectorized_mobject.py (hunk 1 above)
Comment (NeoPlato) [reply manim/mobject/types/vectorized_mobject.py]: I don't have access to a laptop so I was mostly guessing on those format strings.

If you could kindly also help me out with the black accepted format for your fix

**Comment by behackl** [comment]
I've pushed a commit fixing black, and another one making sure that the message is pluralized correctly. Could you cross-check that everything is fine, @NeoPlato?

Then we can move this along.

**Comment by NeoPlato** [comment]
I am totally okay with this @behackl . Merge away!

**Review by behackl — APPROVED** [review]

**Pull request merged**